<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseManagementWidget</class>
 <widget class="QWidget" name="CaseManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/sovajo_logo/icon_images/sovajo_logo/90x28_logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonUser">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/user_static.png);}
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/user_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/user_mouse_click.png); }
QToolButton::menu-indicator{	image: none;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/32px_function_button_icon/icon_images/32px_function_button_icon/user_static.png</normaloff>:/32px_function_button_icon/icon_images/32px_function_button_icon/user_static.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonAdmin">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/menu_static.png); }
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/menu_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/menu_mouse_click.png) ;}
QToolButton::menu-indicator{	image: none;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/32px_function_button_icon/icon_images/32px_function_button_icon/menu_static.png</normaloff>:/32px_function_button_icon/icon_images/32px_function_button_icon/menu_static.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonScreenshot">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{border-image:url(:/svpk_lib/images/system_admin/screenshot_static.png); }
QToolButton:hover{border-image:url(:/svpk_lib/images/system_admin/screenshot_mouse_hover.png); }
QToolButton:pressed {border-image:url(:/svpk_lib/images/system_admin/screenshot_mouse_click.png) ;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonShutDown">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton         {  border-image:url(:/svpk_lib/images/system_admin/shutdown_static.png) ;}
QToolButton:hover   {   border-image:url(:/svpk_lib/images/system_admin/shutdown_mouse_hover.png); }
QToolButton:pressed {   border-image:url(:/svpk_lib/images/system_admin/shutdown_mouse_click.png); }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMid" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="searchInputEdit">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelID">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelImportTime">
          <property name="text">
           <string>Import Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="patientTableView">
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importBtn">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  Import</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_import.png</normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_import.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabDicomExternalData">
        <attribute name="title">
         <string>External Data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
       <widget class="QWidget" name="tabDicomLocalStorage">
        <attribute name="title">
         <string>Local Storage</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5"/>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget2" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QListView" name="listViewScreenshotPics"/>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget3" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelBonePic1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelBonePic2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="labelBonePic3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="labelBonePic4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelBonePic5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labelBonePic6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="labelBonePic7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="labelBonePic8">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="manageBtn">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>  Manage</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/others/icon_images/others/40px_manage_icon.png</normaloff>:/others/icon_images/others/40px_manage_icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectAllBtn">
            <property name="text">
             <string>  Select All </string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/others/icon_images/others/40px_select_all.png</normaloff>:/others/icon_images/others/40px_select_all.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBtn">
            <property name="text">
             <string>  Delete</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_clear.png</normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_clear.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportBtn">
            <property name="text">
             <string>  exportBtn</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_export.png</normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_export.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextStepBtn">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>  Next</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_next_page.png</normaloff>:/40px_button_cutout/icon_images/40px_button_cutout/40px_next_page.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>widgetMid</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources>
  <include location="resource/resource.qrc"/>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
